Python/ast_opt.c

```diff
@@ -268,38 +263,44 @@ fold_subscr(expr_ty node, PyArena *arena)
     return make_const(node, newval, arena);
 }
 
+static int
```

Comment:
It would be nice to add comments and/or a "docstring" to explain what this is doing.